articles/active-directory/develop/msal-net-token-cache-serialization.md

```diff
@@ -247,6 +249,8 @@ You can specify that you don't want to have any token cache serialization and re
        .WithAuthority(authority)
        .Build();
 ```
+\
```

Comment:
```suggestion
```

Reply:
Darn it, I thought I removed that. Fat fingers...